draft-ietf-jsonpath-base.md

```diff
@@ -393,7 +390,7 @@ constant.
 | `$..book[?(@.isbn)]`                      | filter all books with ISBN number                            |
 | `$..book[?(@.price<10)]`                  | filter all books cheaper than 10                             |
 | `$..*`                                    | all member values and array elements contained in input value |
-{: #tbl-example title="Example JSONPath expressions applied to the example JSON value"}
+{: #tbl-example title="Example JSONPath expressions and their intended results"}
```

Comment:
This used to refer to the previous table -- do we want to completely sever that link?

Reply:
Ah, probably not. I was fearful of making the table caption too verbose. I'll add back the link.

Reply:
Fixed.